src/common/metrics/vaccinations.tsx

```diff
@@ -65,13 +65,24 @@ function renderStatus(projections: Projections): React.ReactElement {
   const peopleVaccinated = formatInteger(info.peopleVaccinated);
   const percentVaccinated = formatPercent(info.ratioVaccinated, 1);
 
+  let distributedText = <Fragment />;
+  if (info.dosesDistributed && info.ratioDosesAdministered) {
+    distributedText = (
+      <Fragment>
+        This this means that {locationName} has administered{' '}
+        {formatPercent(info.ratioDosesAdministered)} of their supply of{' '}
```

Comment:
current supply?

Reply:
Yeah, I was thinking about that too.  Done.